src/main/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicy.java

```diff
@@ -138,22 +136,43 @@ public boolean hasActualCostFee() {
     return getActualCostFee().isChargeable();
   }
 
-  public static LostItemPolicy unknown(String id) {
-    return new UnknownLostItemPolicy(id);
+  public boolean shouldChargeFeesWhenAgedToLost() {
+    // i.e. we exclude immediate billings that have nothing to charge
+    return shouldDelayBillingForPatronWhenItemAgedToLost()
+      || setCostFee.isChargeable() || ageToLostProcessingFee.isChargeable();
   }
 
   public boolean canAgeLoanToLost(DateTime loanDueDate) {
-    if (agedToLostAfterOverdueInterval == null
-      || agedToLostAfterOverdueInterval.toMinutes() <= 0) {
+    if (agedToLostAfterOverdueInterval.isEmpty()) {
```

Comment:
Is there any distinction between the period not being defined and it being zero length?

Reply:
No, if the period is not defined than it is defaulted to the `emptyPeriod()`.